src/github/graphql.ts

```diff
@@ -117,15 +120,26 @@ export interface Commit {
 	id: string;
 	commit: {
 		author: {
+			avatarUrl: string
+			name: string
+			email: string
```

Comment:
It's possible for the author and/or committer to not be _GitHub_ users, but rather just standard git users. In those cases the email will come from the commit, rather than the GitHub profile.